src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Monaco.Edit.cshtml

```diff
@@ -21,65 +21,71 @@
 
 <script asp-name="monaco" at="Foot"></script>
 <script at="Foot" depends-on="monaco">
-    document.addEventListener('DOMContentLoaded', function() {
+    document.addEventListener('DOMContentLoaded', function () {
         require(['vs/editor/editor.main'], function () {
 
-            var settings = @Html.Raw(monacoSettings.Options);
-            if (settings.automaticLayout == undefined) {
-                settings.automaticLayout = true;
-            }
-            settings.language = 'html';
+            var divEditor = document.getElementById('@Html.IdFor(x => x.Html)_editor');
```

Comment:
you can change `var` to `const` and eliminate the `if (divEditor != undefined || divEditor != null) {` check.


or if you don't want to use `const`, then do this instead

```
if (!divEditor) {
    return;
}
```

which should give you the same result but much less indentation.